graql/reasoner/arithmetic-equality.feature

```diff
@@ -0,0 +1,277 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+
+#  NOTE
+#  This file is retained as a placeholder for future functionality
+#  The following tests can no longer be run due to restrictions on HAS constraints in then clauses.
+#  The ability to take the value of one attribute and give it to another will, however, be useful
+#  Therefore every test here will be ignored until further change.
```

Comment:
Um, is arithmetic equality the right name for this behaviour?

Reply:
well it means using arithmetic to make two values equal. So the file name is speculative of how arithmetic will eventually be implemented, and will probably change. The file is really just a place to keep tests in the meantime.

Reply:
Ok seems reasonable